package.json

```diff
@@ -18,6 +18,8 @@
   "dependencies": {
     "@gnosis.pm/safe-apps-react-sdk": "4.0.5",
     "@gnosis.pm/safe-apps-sdk": "5.0.0",
+    "@gnosis.pm/safe-react-components": "git://github.com/gnosis/safe-react-components.git#27ecf08b4a9fbd6d81be6d550bbdfc92eb632d68",
```

Comment:
We should be careful with this move. Currently all safe apps are using different versions of `safe-react-components`. If we move this here an update could lead to a full check of all the other apps

Reply:
In general, I think we should restrict this practice of using git commit hashes to development only. If you remove this hash from GitHub (let's say remove a branch), you won't install dependencies anymore.